docs/src/main/asciidoc/cache.adoc

```diff
@@ -288,7 +288,8 @@ If the method has one or more arguments, the key computation is done from all th
 This annotation can also be used on a method with no arguments, a default key derived from the cache name is generated in that case.
 If the key does not identify any cache entry, nothing will happen.
 
-You can only use one of the cache operations (and this annotation) on a given method: `@CacheResult`, `@CacheInvalidate` or `@CacheInvalidateAll`.
+This annotation can be combined with multiple other caching annotations on a single method.
```

Comment:
I wonder if you need to mention that, do you think it helps the reader or is it just a remnent of the old implementation that you want to emphasize the change of?

Reply:
The first sentence is only there to introduce the second one about the execution order. I think the execution order is an important detail as it allows the user to force a cache value refresh for example (by combining `@CacheInvalidate` and `@CacheResult`). But maybe I should go one step further and write down that specific example in the doc.